### ISSUE TITLE
Package as Flatpak for Flathub?

### ISSUE THREAD

**Issue by cassidyjames** [timmo001/home-assistant-assist-desktop#66]
It'd be great to get this as a Flatpak so it can be trivially discovered and installed across all Linux desktops—including in the Steam Deck desktop's app store. :)

**Comment by timmo001** [comment]
Looks like this has been a topic of discussion for a while on tauri, so we would have to wait on the tauri project for support:

https://github.com/tauri-apps/tauri/issues/3619

**Comment by timmo001** [comment]
None, its likely that the tauri team are focused on getting v2 across the line. Unless a plugin gets created to extend the build tools, I don't think we'll see this for a while

**Comment by Korne127** [comment]
@IoSonoAndreaZ The AppImage issue should be fixed now as https://github.com/AppImageCommunity/pkg2appimage/pull/555 is merged (when building a new version).

**Comment by IoSonoAndreaZ** [comment]
> @IoSonoAndreaZ The AppImage issue should be fixed now as [AppImageCommunity/pkg2appimage#555](https://github.com/AppImageCommunity/pkg2appimage/pull/555) is merged (when building a new version).

thank you, should I wait for a new appimage build? the latest is still the 1.4.0

**Comment by Korne127** [comment]
Yes, this isn't fixed without a new AppImage. Additionally, the project maintainers need to wait for a `tauri` update until this is resolved here.
